src/components/users-avatar-list.component.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  StyleSheet,
+  View,
+  Text,
+  FlatList,
+  TouchableHighlight,
+  Image,
+} from 'react-native';
+import { List, ListItem } from 'react-native-elements';
+
+import { colors, fonts } from 'config';
+
+const mapStateToProps = state => ({
+  authUser: state.auth.user,
+});
+
+type Props = {
+  title: string,
+  members: Array,
+  noMembersMessage: string,
+  containerStyle: Object,
+  smallTitle: string,
+  navigation: Object,
+  authUser: Object,
+  loadMore: Function,
+};
+
+const size = 30;
+
+const styles = StyleSheet.create({
+  container: {
+    marginTop: 30,
+  },
+  avatarContainer: {
+    backgroundColor: colors.greyLight,
+    borderRadius: size / 2,
+    width: size,
+    height: size,
+    marginRight: 5,
+  },
+  avatar: {
+    borderRadius: size / 2,
+    height: size,
+    width: size,
+  },
+  list: {
+    marginTop: 0,
+  },
+  sectionTitle: {
+    color: colors.black,
+    ...fonts.fontPrimaryBold,
+    marginBottom: 10,
+    paddingLeft: 15,
+  },
+  sectionTitleSmall: {
+    color: colors.primaryDark,
+    ...fonts.fontPrimarySemiBold,
+    marginBottom: 10,
+    paddingLeft: 15,
+  },
+  flatList: {
+    paddingLeft: 15,
+    paddingRight: 15,
+  },
+});
+
+const UsersAvatarListComponent = ({
```

Comment:
Partially the last avatar is not displayed:

![image](https://user-images.githubusercontent.com/4408379/31441903-45f754ac-ae9d-11e7-9135-e87f63bec856.png)

Reply:
Yes, that's the case in `master` too